pkgs/development/tools/ldid/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, callPackage
+, fetchgit
+, libplist
+, libxml2
+, openssl_1_1
```

Comment:
why does this explicitly use `openssl_1_1`? seems to build just fine with `openssl`.

Reply:
Let's migrate to `openssl` then. Would you like to open the PR?

Reply:
done #222378